### PR TITLE
Add reset filters timer button to Smarty .markdown

### PR DESCRIPTION
**Pull request by marcodutto**
Smarty has the possibility to reset the filters timer when filters are changed. This function can now be used from HA

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/129637
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Added a new category "Button" to the integration documentation for Salda Smarty ventilation systems.
	- Introduced "button" as a new platform alongside existing platforms.
	- Expanded integration capabilities to include the ability to reset the filter's timer via the button platform.
	- Updated the list of supported device types for enhanced clarity.

### CODE REVIEW

File: source/_integrations/smarty.markdown [hunk 1]
```diff
@@ -22,10 +24,11 @@ The `smarty` integration lets you control Salda [Smarty](http://www.salda.lt/en/
 
 There is currently support for the following device types within Home Assistant:
 
+- Button
 - Fan
 - Sensor
 
-The integration has a fan platform to view and control the ventilation speed, and a sensors platform to read:
+The integration has a fan platform to view and control the ventilation speed, a button platform to reset the filters timer and a sensors platform to read:
```
Comment (c0ffeeca7) [inline source/_integrations/smarty.markdown]:
```suggestion
The integration has a fan platform to view and control the ventilation speed, a button platform to reset the filter's timer, and a sensors platform to read the following values:
```

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @marcodutto :+1:

**Comment by c0ffeeca7** [comment]
A merge conflict appeared. Could you take a look, @marcodutto?

**Review by frenck — CHANGES_REQUESTED** [review]
Hi there @marcodutto! 👋 

The parent PR has been merged, but this documentation PR has a merge conflict. Could you please take a look and resolve it?

Thanks! 👍 

../Frenck

**Review by frenck — APPROVED** [review]
Thanks, @marcodutto 👍 

../Frenck

**Pull request merged**